dataservice/api/genomic_file/models.py

```diff
@@ -0,0 +1,36 @@
+from dataservice.extensions import db
+from dataservice.api.common.model import Base
+
+# TODO May want to change all uuid strings postgres UUID later on
+
+
+class GenomicFile(db.Model, Base):
+    """
+    GenomicFile entity.
+
+    :param _id: Unique id assigned by RDBMS
+    :param kf_id: Unique id given by the Kid's First DCC
+    :param created_at: Time of object creation
+    :param modified_at: Last time of object modification
+    :param uuid: UUID assigned to file from Gen3
+    :param file_name: Name of file
+    :param file_type: Type of genomic file (i.e. aligned reads)
+    :param file_format: Format of file
+    :param file_url: Location of file
+    :param md5sum: 128 bit md5 hash of file
+    """
+
+    __tablename__ = 'genomic_file'
+    uuid = db.Column(db.String(36), unique=True)
```

Comment:
What used to be dos_id or gen3 uuid. Per @allisonheath we are using uuid to store the gen3 uuid for now.

Reply:
Can't this just use the uuid from the `Base`?
I think we'll probably generate the uuid inside our api here, then register that uuid with gen3, not the other way around.

Reply:
👍